Scenarios/Logging/LoggingSample/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Project created using the CoreWCF.Templates project template
+
+var builder = WebApplication.CreateBuilder();
+
+builder.Services.AddServiceModelServices();
+builder.Services.AddServiceModelMetadata();
+builder.Services.AddSingleton<IServiceBehavior, UseRequestHeadersForMetadataAddressBehavior>();
+
+//Register the service class with Dependency Injection
+builder.Services.AddSingleton<Service>();
```

Comment:
@mconnew - I have assumed the singleton pattern here - AFAIK there are attributes for other service class patterns - are they worth talking about here?